README.md

```diff
@@ -83,7 +83,23 @@ If everything was successful, you should now have a running stack with an HTTPS
 
 To send your first request, use the `client` script with the GET endpoint URL:
 ```
-npm run client -- https://30r00qsyhf.execute-api.us-east-1.amazonaws.com/lmorchard/accept
+npm run client
```

Comment:
This bit of the README is the basic effect of this PR - should try to auto-configure everything in development to call mock endpoints and let you tail the logs of the functions to see things working